### DISCUSSION TITLE
What is the H-STATE partition

### DISCUSSION THREAD

**Discussion by myaroschuk** [ading2210/shimboot#206] [Q&A]
Been figuring this out as i go along until now, H-state has 45 GB it's not using, and are inaccessible to me, any idea on how to change that?

**Comment by IvanVerdugo** [comment]
js partition it back?

**Comment by Yus4F-07** [comment]
well i deleted the chromeos partitions and the h-state partition, the h-state partition is where all the chromeos data is stored on, the safe partitions to delete and make a new large partition out of it is the ROOT-A, ROOT-B and H-STATE partitions if you don't want chromeos and want to use the storage for something, i use mine for steam games and other files
